core/src/main/resources/Samples/Input_Json_Specification.json

```diff
@@ -418,6 +423,20 @@
         "options": {
           "postactions": "DROP SCHEMA \"DEMO_DB\".\"TESTSCHEMA1\";"
         }
+      },
+      "destination_sftp": {
+        "platform": "sftp",
+        "host": "htp hostname/IP address",
+        "path": "somepath/somefilename.json",
+        "login": "ftpusername",
+        "comment_password": "Optional; password for ftpusername. This attribute is optional/ignored if the pemfilepath attribute is specified.",
+        "password": "ftppassword",
+        "comment_pemfilepath": "Optional; local file path for the PEM RSA private key file for ftpusername. This attribute is not needed if the password attribute is specified. The password attribute is optional/ignored if the pemfilepath attribute is specified. When running on EMR, the jksfilepath should be specified as the S3 location of the PEM file in the format s3://<S3 bucket and folder path>/<PEM file name>, and the pemfilepath should be of the format /mnt/bootstrapfiles/<PEM file name>",
+        "pemfilepath": "/mnt/bootstrapfiles/<PEM file name>",
+        "comment_jksfilepath": "Optional; When running on EMR, the jksfilepath should be specified as the S3 location of the PEM file in the format s3://<S3 bucket and folder path>/<PEM file name>, and the pemfilepath should be of the format /mnt/bootstrapfiles/<PEM file name>",
+        "jksfilepath": "s3://<S3 bucket and folder path>/<PEM file name>",
```

Comment:
I don't see this variable is being used as part of the implementation.  

we can remove this from the documentation.